packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -216,12 +216,20 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
 
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
   const onRenderInlineAttachment = useCallback(
-    async (attachment: FileMetadata): Promise<AttachmentDownloadResult[]> => {
-      if (attachment.attachmentType === 'inlineImage' && attachment.previewUrl) {
-        const blob = await adapter.downloadAttachments({ attachmentUrls: [attachment.previewUrl] });
-        return blob;
-      }
-      return [{ blobUrl: '' }];
+    async (attachment: FileMetadata[]): Promise<AttachmentDownloadResult[]> => {
+      const targets = attachment.filter((obj) => {
+        return obj.attachmentType === 'inlineImage' && obj.previewUrl;
+      });
```

Comment:
Seems like this filter function is unnecessary since in the forEach loop below, we can just loop through the attachment instead of the targets.